### PR TITLE
Feature/react 7 list recognitions

### PR DESCRIPTION
**Pull request by avi4630**

### CODE REVIEW

File: react-frontend/package.json [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
   "version": "0.1.0",
   "private": true,
-  "license" : "MIT",
+  "license": "MIT",
   "description": "Peerly - Rewards and recognition system",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
+    "@reach/menu-button": "^0.10.0",
```
Comment (sahilbhatia) [inline react-frontend/package.json:11]: @avi4630 - why do we need these packages?

File: react-frontend/package.json [hunk 2]
```diff
@@ -2,17 +2,23 @@
...
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.4.1",
     "react": "^16.13.1",
     "react-bootstrap": "^1.0.0",
     "react-dom": "^16.13.1",
+    "react-fontawesome": "^1.7.1",
```
Comment (sahilbhatia) [inline react-frontend/package.json:19]: @avi4630 - why do we need this package?

File: react-frontend/package.json [hunk 3]
```diff
@@ -2,17 +2,23 @@
...
     "react-scripts": "3.4.1",
+    "reactstrap": "^8.4.1",
```
Comment (sahilbhatia) [inline react-frontend/package.json:21]: Please look at existing code also. When we already have `react-bootstrap`, why do we need to add `reactstrap` package?

File: react-frontend/src/App.js [hunk 4]
```diff
@@ -1,24 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ListRecognition></ListRecognition>
```
Comment (sahilbhatia) [inline react-frontend/src/App.js:8]: Components that do not have child, should have self-closing tags

File: react-frontend/src/components/listRecognitions.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Row, Col, Label, Button, Container } from "reactstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (sahilbhatia) [inline react-frontend/src/components/listRecognitions.js:3]: Please remove this line.

1. We already have imported bootstrap CSS at a common place.
2. Every component should not include bootstrap CSS

File: react-frontend/src/components/listRecognitions.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Row, Col, Label, Button, Container } from "reactstrap";
```
Comment (sahilbhatia) [inline react-frontend/src/components/listRecognitions.js:2]: Check my comment  (in package.json) on using "react-bootstrap"

File: react-frontend/src/components/listRecognitions.js [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+import {
+  MenuButton,
+  MenuList,
+  MenuItem,
+  MenuLink,
+  Menu,
+} from "@reach/menu-button";
+import "@reach/menu-button/styles.css";
+const ListRecognition = () => {
+  const persons = [
```
Comment (sahilbhatia) [inline react-frontend/src/components/listRecognitions.js:15]: Why are we hard-coding data?

File: react-frontend/src/components/listRecognitions.js [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+        "recognition text:welcome to bonusly @sahil+bonasuly ! we are recognistion you with +25",
+      core: "welcome to bonusly",
+    },
+  ];
+  return (
+    <Container>
+      <Row>
+        <Col xs="3"></Col>
+        <Col>
+          {persons.map((person) => (
```
Comment (sahilbhatia) [inline react-frontend/src/components/listRecognitions.js:46]: Do not use `inline functions` in render. Move the code inside this loop to a separate `Person` component.

File: react-frontend/package.json (hunk 2 above)
Comment (avi4630) [reply react-frontend/package.json:19]: for 3 dot menu icon

**Pull request closed without merging**